app/views/content/life-as-a-teacher/teaching-as-a-career/turning-a-tough-lesson-into-a-success.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Turning a tough lesson into a success
+title_paragraph: |-
+  <p>PE teacher Danny shares how he tackles challenging behaviour in PE lessons.</p>
+description: |-
+  PE teacher Danny shares how he tackles challenging behaviour in PE lessons.
```

Comment:
This meta has PE twice - does it need changing?

(Also see previous comment re PE vs physical education - could maybe have both here if it would be helpful re SEO?)